JS/index.js

```diff
@@ -1,9 +1,48 @@
-const libraryBooksEl = document.querySelector('.library-books');
+const listEl = document.querySelector('.list');
+const addnewEl = document.querySelector('.addnew');
+const contactEl = document.querySelector('.contact');
+const date = document.querySelector('.date');
+const booksListEl = document.querySelector('.awesome-book');
+const libraryBooksEl = document.querySelector('.list-books');
 const addElBtn = document.querySelector('.btn-add');
+const contactInformEl = document.querySelector('.contact-inform');
+const form = document.querySelector('form');
 const titleEl = document.getElementById('title');
 const authorEl = document.getElementById('author');
-const errorMsgEl = document.querySelector('.errorMsg');
-const form = document.querySelector('form');
+const alertMessageEl = document.querySelector('.alertMessage');
+setInterval(() => {
+  date.innerHTML = new Date();
+}, 1000);
+
+const toggleWindow = () => {
+  listEl.addEventListener('click', () => {
+    form.classList.remove('show');
+    contactInformEl.classList.remove('show');
+    booksListEl.classList.remove('hide');
+    listEl.style.color = 'blue';
+    addnewEl.style.color = 'black';
+    contactEl.style.color = 'black';
+  });
+
+  addnewEl.addEventListener('click', () => {
+    form.classList.add('show');
+    contactInformEl.classList.remove('show');
+    booksListEl.classList.add('hide');
+    listEl.style.color = 'black';
+    addnewEl.style.color = 'blue';
+    contactEl.style.color = 'black';
+  });
+
+  contactEl.addEventListener('click', () => {
+    form.classList.remove('show');
+    contactInformEl.classList.add('show');
+    booksListEl.classList.add('hide');
+    listEl.style.color = 'black';
+    addnewEl.style.color = 'black';
+    contactEl.style.color = 'blue';
+  });
+};
+
```

Comment:
- Please note that I couldn't add a new book to the list from the form, kindly fix this 👍🏽

Reply:
@rotimiazeez sorry I don't understand you

Reply:
@GraceAbike when I try to add a book to the list of books, I get no response.

Reply:
Yes you are right thanks, I have fixed it